packages/integrations/useCookies/index.ts

```diff
@@ -0,0 +1,103 @@
+import { ref, getCurrentInstance, watchEffect } from 'vue-demi'
+import Cookie from 'universal-cookie'
+import type { IncomingMessage } from 'http'
+
+type RawCookies = {
+  [name: string]: string
+}
+
+let universalCookie: Cookie | undefined
+
+/**
+ * Initializing universal-cookie using request (default is window.document.cookie)
+ * @param {Object} req - incoming http request (for SSR)
+ * @see {@link https://github.com/reactivestack/cookies/tree/master/packages/universal-cookie|universal-cookie}
+ */
+export const createCookies = (req?: IncomingMessage) => {
+  universalCookie = new Cookie(req ? req.headers.cookie : null)
+
+  return universalCookie
+}
+
+/**
+ * Reactive methods to work with cookies (firstly call {@link createCookies} method if using SSR)
+ * @param {string[]|null|undefined} dependencies - array of watching cookie's names. Pass empty array if don't want to watch cookies changes.
+ * @param {boolean} [doNotParse=false] - don't try parse value as JSON
+ */
+export const useCookies = (dependencies?: string[] | null, doNotParse = false) => {
```

Comment:
For exported functions, I would prefer `function` over arrow functions.

Reply:
Mostly for tree-shaking sake. Variables without `/*#__PURE__*/` will be treat as side effects and not tree-shakable by default. Although bundles should be intelligent enough to tell it's an arrow function, using the `function` keyword feels more explicit to me.